kubernetes/deployment/deployment.yaml

```diff
@@ -22,7 +22,7 @@ spec:
           #- --kubeconfig=kubeconfig.yaml ## Optional parameter used when controller is running outside the cluster, kubeconfig.yaml file can be fetched from Kubernetes config-map or secret
         livenessProbe:
           httpGet:
-            path: /livez
+            path: /s/livez/healthz
```

Comment:
What meant was a replace `livez` with `healthz`. I was just using the common sed notation for the same. Sorry about that. Can you quickly just change it `/s/livez/healthz` to `/heathlz`